src/Employer/Employer.Web/appsettings.json

```diff
@@ -21,7 +21,7 @@
     "ClientSecret": ""
   },
   "GoogleAnalytics": {
-    "TrackingId": "UA-83918739-16"
+    "TrackingManagerCode": "GTM-W7DMV8F"

```

Comment:
Should this be in our repo?

Reply:
I don't suppose it matters too much as it's on the actual page isn't it?

Reply:
Yeah it's visible.

Reply:
Of course!